Folge_02/index.html

```diff
@@ -0,0 +1,16 @@
+<html>
+
+<head>
+  <title>My first react application</title>
```

Comment:
```suggestion
  <title>My first React application</title>
```

React ist ja ein Produktname und wird mit großem R geschrieben.